plugins/core/Standard/qPCL/PclUtils/filters/BaseFilter.h

```diff
@@ -184,7 +184,7 @@ class BaseFilter: public QObject
 	//! Called when action is triggered
 	/** \note performAction calls start() in base class
 	**/
-	int performAction();
+	Q_SLOT int performAction();
```

Comment:
As discussed in Slack, the correct solution is to convert the connects to Qt5-style connects.

Reply:
Has it been done? If not, I just want to make this plugin compile for now...

Reply:
Can't you just change the connects instead?

Reply:
Don't really have the time for that (I'm struggling like hell to rebase my most important branch right now with all the recent changes)

Reply:
I would do it, but I can't compile them to make sure I get it right.

My IDE has an option that just converts old-style to new style - does that exist in [QtVisualStudioTools2019](https://marketplace.visualstudio.com/items?itemName=TheQtCompany.QtVisualStudioTools2019)?

Reply:
I don't use the Qt Visual addin anymore (because it was kind of 'conflicting' with CMake and causing strange issues). Not very keen to go back to it. And I clearly don't have the time for that now.

Reply:
> it was kind of 'conflicting' with CMake 

Ah - well Qt6 is going to be CMake-based, and they've already made a lot of changes to support cmake with Qt Creator, so they're moving everything that way!